sphinxcontrib/doxylink/__init__.py

```diff
@@ -1,10 +1,13 @@
-__version__ = "1.12.3"
+from . import config
 
+__version__ = "1.12.3"
 
 def setup(app):
     from .doxylink import setup_doxylink_roles
     app.add_config_value('doxylink', {}, 'env')
     app.add_config_value('doxylink_pdf_files', {}, 'env')
+    app.add_config_value(config.DOXYLINK_IGNORE_PARSE_EXCEPTIONS_KEY,
```

Comment:
I will add this configuration to the _index.rst_ documentation.

Reply:
@JasperCraeghs the only reason I added that was so I didn't have to hardcode the configuration key for ignoring parse exceptions. I am totally cool with removing that and hardcoding per your recommendation.